src/main/java/dk/cngroup/lentils/controller/ClientController.java

```diff
@@ -0,0 +1,34 @@
+package dk.cngroup.lentils.controller;
+
+import dk.cngroup.lentils.service.CypherGameInfoService;
+import dk.cngroup.lentils.service.TeamService;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/")
+public class ClientController {
+
+    private static final String CLIENT_VIEW_CYPHER_LIST = "client/cypher/list";
+
+    private CypherGameInfoService cypherGameInfoService;
+    private TeamService teamService;
+
+    @Autowired
+    public ClientController(final CypherGameInfoService cypherGameInfoService,
+                            final TeamService teamService) {
+        this.cypherGameInfoService = cypherGameInfoService;
+        this.teamService = teamService;
+    }
+
+    @GetMapping(value = "cyphers")
+    public String listAllCyphers(final Model model) {
+        model.addAttribute("cypherGameInfos", cypherGameInfoService.getAll());
+        model.addAttribute("team", teamService.getTeam(2L));
```

Comment:
Proc 2L?

Reply:
To je dočasné řešení, než se vyřeší team, který je logged in.